projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -39,11 +38,21 @@ const ProtectValueSection = () => {
 			}
 			return null;
 		}
-		return sprintf(
-			/* translators: `\xa0` is a non-breaking space. %1$d is the number (integer) of plugins and %2$d is the number (integer) of themes the site has. */
-			__( '%1$d plugins &\xa0%2$d\xa0themes', 'jetpack-my-jetpack' ),
-			pluginsCount,
-			themesCount
+		return (
+			sprintf(
+				/* translators: %d is the number of plugins installed on the site. */
+				_n( '%d plugin', '%d plugins', pluginsCount, 'jetpack-my-jetpack' ).replace( ' ', '\xa0' ), // `\xa0` is a non-breaking space.
```

Comment:
Thanks @robertsreberski but the goal here isn't to keep everything on one line, I'm just preventing the line from breaking between then number/digit and the following text, for example I don't want the text "5 plugins & 3 themes" to break between "5" and "plugins" or between "3" and "themes".  That way if it has to break into two lines, it will break like this:
5 plugins
& 3 themes

but not:
5 plugins & 3
themes

Anyway, that was the intention here and the reason for the specifically placed non-breaking spaces.
Although realizing now I could probably remove the non-breaking space between "5" and "plugins" (and keep the nbsp between "3" and "themes") because I don't think the card will ever squeeze small enough that it would wan't to break after the very first number, like between "5" and "plugins" in the text "5 plugins & 3 themes".